transfers/views/student_views.py

```diff
@@ -105,6 +124,21 @@ def post(self, request, *args, **kwargs):
             )
         hod_email_list = list(hod_email_qs)
         self.context = {'form': form, 'hod_email_list': hod_email_list}
+#Notification System
+        body = ""
+        for data in TS2PSTransfer.objects.all():
+            if request.user == data.applicant.user:
+                body =  str("\nID: " + data.applicant.user.username+
+                    "\nName: " + data.applicant.user.first_name + " " +data.applicant.user.last_name +
+                    "\nTransfer Type: " + data.get_sub_type_display()+
+                    "\nCGPA: " + str(data.cgpa)+
+                    "\nReason For Transfer: " + data.reason_for_transfer+
+                    "\nOrganization Name: " + data.name_of_org)
+                print("done")
+                send_mail("Transfer Application: "  + request.user.username, body,
+                    'psdmail@gmail.com',[str(data.hod_email)],
```

Comment:
If possible, can you move this entire block of code into a new function in `utils.py`? And call that function from here. Also,is it really needed to iterate the `TS2PSTransfer` table (I am referring to `for data in TS2PSTransfer.objects.all()`)?

Same remarks apply to the above code blocok.

Reply:
fixed